tools/dgeni/templates/property.template.html

```diff
@@ -19,7 +21,9 @@
       </div>
     {%- endif -%}
     {%- if property.isDeprecated -%}
-      <div class="docs-api-deprecated-marker">Deprecated</div>
+      <div class="docs-api-deprecated-marker" {$ macros.deprecationTitle(property) $}>
```

Comment:
It would also be cool to add a dotted underline to the deprecation marker to indicate that there's a hover effect. We could do this in a follow-up PR on the docs site w/ something like
```scss
.docs-api-deprecated-marker[title] {
  border-bottom: 1px dotted gray;
}
```